doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,96 @@ The query returns all the entities that match all the matching conditions (AND l
 
 ```
 temperature==50
 temperature<=20
 ```
 
+The attribute path is used to select the *element under evaluation* for the statement. Its general
+syntax consists of an attribute name optionally followed by a *navigation path* using the `.`
+character as delimiter, e.g. "temperature.accuracy.average" refers to the navigation path "accuracy.average"
+within the "temperature" attribute.
+
+How the navigation path is interpreted depends on where the Simple Query Language statement is used:
+
+* If used in a `q` query, then the navigation path refers to a sub-key of the attribute value
+  (assuming that the attribute value is a JSON object). If no navigation path is provided (i.e.
+  the attribute path doesn't have any `.`) then the attribute itself is taken into account.
+* If used in a `mq` query, then the first element of the navigation path is mandatory and it means the
+  name of a metadata of the attribute. Additional elements in the navigation path refer to a sub-key of
+  the metadata value (assuming that the metadata value is a JSON object). If no navigation path is provided
+  (beyond the first element which identifies the metadata name) then the metadata itself
+  is taken into account.
+
 The list of operators (and the format of the values they use) is as follows:
 
 + **Equal**: `==`. This operator accepts the following types of right-hand side:
-    + Single element, e.g. `temperature==40`. For an entity to match, it must contain an attribute
-      with that exact name (temperature) and value (40).
+    + Single element, e.g. `temperature==40`. For an entity to match, it must contain an element
+      under evaluation with that exact name (temperature) and value (40) (or which includes the
+      value, in the case of elements under evaluation which value is an array).
     + A list of comma-separated values, e.g. `color==black,red`. For an entity to match, it must
-      contain an attribute with that name and the value of the attribute must be **any** of the
-      values in the list (OR clause).
+      contain an element under evaluation with that name and its value must be **any** of the
+      values in the list (OR clause) (or which includes **any** of the values in the case of
+      elements under evaluation which value is an array).
       E.g. entities with an attribute named `color`, whose value is `black` are a match, while
       entities with an attribute named `color` but whose value is `white` do not match.
     + A range, specified as a minimum and a maximum, separated by `..`, e.g. `temperature==10..20`.
-      For an entity to match, it must contain an attribute with that name (temperature), and its
-      value must be in the interval between the minimum and maximum of the range (both included).
-      Ranges can only be used with attributes that represent dates (in ISO8601 format), numbers or
-      strings.
+      For an entity to match, it must contain an element under evaluation with that name (temperature),
```

Comment:
In this phrase 'element under evaluation' sounds really strange ...
Just 'element' or 'matching element' would be a lot better